### PR TITLE
GH-17 Add okaeri serializer and test for it.

### PR DESCRIPTION
**Pull request by vLuckyyy**
Fixes: GH-17

If you consider this to be a needed, we can add unit test for the MultificationNoticeSerializer. As long as awesome helps me with this.

test coverage:
image

### CODE REVIEW

**Review by CitralFlo — APPROVED** [review]

**Comment by CitralFlo** [comment]
I am wondering if GH actions runs all testes, because I couldn't find any mention in logs, except
`> Task :multification-okaeri:test`

File: multification-okaeri/src/com/eternalcode/multification/okaeri/MultificationNoticeSerializer.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    public Notice deserialize(DeserializationData data, @NotNull GenericsDeclaration generics) {
+
+        Builder builder = Notice.builder();
+
+        Set<String> keys = data.asMap().keySet();
+
+        if (keys.size() == SINGLE_SERIALIZE_DESERIALIZE_PART && data.isValue()) {
+            Object value = data.getValueRaw();
+
+            if (value instanceof String) {
```
Comment (imDMK) [inline multification-okaeri/src/com/eternalcode/multification/okaeri/MultificationNoticeSerializer.java:95]:
```suggestion
            if (value instanceof String stringValue) {
```

File: multification-okaeri/src/com/eternalcode/multification/okaeri/MultificationNoticeSerializer.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+                List<String> messages = Collections.singletonList((String) value);
```
Comment (imDMK) [inline multification-okaeri/src/com/eternalcode/multification/okaeri/MultificationNoticeSerializer.java:96]:
```suggestion
                List<String> messages = Collections.singletonList stringValue);
```

File: multification-okaeri/src/com/eternalcode/multification/okaeri/MultificationTimesTransformer.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    public GenericsPair<String, Times> getPair() {
+        return this.genericsPair(String.class, Times.class);
+    }
+
+    @Override
+    public Times leftToRight(String data, SerdesContext serdesContext) {
+        String[] split = data.split(" ");
+
+        if (split.length != 3) {
+            throw new IllegalArgumentException(
```
Comment (imDMK) [inline multification-okaeri/src/com/eternalcode/multification/okaeri/MultificationTimesTransformer.java:27]: I suggest returning the default values. For example: 1s 2s 1s

**Review by imDMK — APPROVED** [review]
Good job.

**Pull request merged**